plugins/ScheduledReports/API.php

```diff
@@ -514,6 +513,7 @@ public function generateReport(
             Config::setSetting('General', 'graphs_default_evolution_graph_last_days_amount', $originalDefaultEvolutionGraphLastPeriodsAmount);
 
             // restore filter truncate parameter value
+            $initialFilterTruncate = Common::getRequestVar('filter_truncate', false);
```

Comment:
I think this should be moved before the whole try/catch block. Otherwise the `$_GET['filter_truncate'] = ...` within the try, might overwrite it.

Reply:
Done :+1: